lib/Sema/CSFix.h

```diff
@@ -58,8 +58,6 @@ enum class FixKind : uint8_t {
 
   /// Introduce a '&' to take the address of an lvalue.
   AddressOf,
-  /// Remove extraneous use of `&`.
-  RemoveAddressOf,
```

Comment:
Although this is a contextual failure, let's keep it's special kind since we know what it is.